app/controllers/v1/reservations_controller.rb

```diff
@@ -0,0 +1,68 @@
+class V1::ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[show edit update destroy]
+
+  # GET /reservations or /reservations.json
+  def index
+    @user = current_user
+    @reservations = Reservation.where(user_id: @user.id)
```

Comment:
- [ ] This could be done in a more efficient way, `where` is a very expensive query. Shouldn't the `user` have many reservations in this case? If yes, you could simply use `@user.reservations`, right?